src/algebra/free_non_unital_non_assoc_algebra.lean

```diff
@@ -42,7 +42,7 @@ noncomputable theory
 variables (R : Type u) (X : Type v) [semiring R]
 
 /-- The free non-unital, non-associative algebra on the type `X` with coefficients in `R`. -/
-@[derive [inhabited, non_unital_non_assoc_semiring, module R]]
+@[derive [inhabited, non_unital_non_assoc_semiring]]
 def free_non_unital_non_assoc_algebra := monoid_algebra R (free_magma X)
```

Comment:
If we made this an `abbreviation` then none of the typeclasses below would need copying.

Reply:
No reason, `abbreviation` probably is the better choice.

Reply:
Fixed